apigee_m10n.permissions.yml

```diff
@@ -1,9 +1,6 @@
 administer apigee monetization:
   title: 'Administer apigee monetization'
 
-access monetization packages:
-  title: 'Access monetization packages'
-
 view mint prepaid reports:
   title: 'View Mint Prepaid Reports'
```

Comment:
We should probably create a ticket and rename this to "view prepaid balance". Not sure why this is using Mint!

Reply:
There is a ticket to look at the permissions. [DRUP-555]